assets/app/view/create_game.rb

```diff
@@ -95,7 +95,11 @@ def submit
         players = params
           .select { |k, _| k.start_with?('player_') }
           .values
-          .map { |name| { name: name } }
+          .map { |name| name.gsub(/\s+/, ' ').strip }
+
+        if players.any? { |name| players.count(name) > 1 }
```

Comment:
i think this could be simplified as if players.uniq.size < players.size

Reply:
return store(:flash_opts) if player.uniq.size < players.size

Reply:
Sounds like you have a one line fix for #71 in mind. I think it'll save us both time if you commit that directly to master, rather than me updating this PR.

Reply:
i will just merge this and update it

Reply:
Thank you Toby! 🚂

Reply:
thanks for your first PR, looking forward to your log fixes